### ISSUE TITLE
Write Errors to log file

### ISSUE THREAD

**Issue by Testspieler09** [Testspieler09/focus_timer#4]
I think that when using curses it is better to have a file that tracks the errors and (debug) prints.
Thats why a **log function** would be great.

It should redirect the IO stream to the log file.

```python
def log(level: str, message: str) -> None:
       pass
```
A sample output would be `[ERROR] Something went wrong when playing the sound file at ...`

I wont work on this for this project but might do it for other programs that use curses in the future.

Thanks to anyone that will contribute to this in the future

**Comment by MiRiskyy** [comment]
Here is one way to implement the log function:

```python
import logging
import sys

LOG_LEVELS = {'debug': logging.DEBUG, 
              'info': logging.INFO,
              'warning': logging.WARNING,
              'error': logging.ERROR,
              'critical': logging.CRITICAL}

def setup_logging(log_file):
    formatter = logging.Formatter('[%(levelname)s] %(message)s')
    handler = logging.FileHandler

**Comment by Testspieler09** [comment]
Thank you for pointing me to the logging package/library. 👀
The provided code is a bit confusing (why import sys and define the log_levels) but all in all it helped a lot. 👍
I implemented the logging func right now ([85f5a3e](https://github.com/Testspieler09/focus_timer/commit/85f5a3e17677767e95b4a7b500fe5a85be8a1caf)) as it was that simple.
